qa/check-docstyle.sh

```diff
@@ -18,11 +18,15 @@ fail=0
 function prepare_venv() {
     VIRTUALENV="$(which virtualenv)"
     if [ $? -eq 1 ]; then
-        # python36 which is in CentOS does not have virtualenv binary
+        # python34 which is in CentOS does not have virtualenv binary
         VIRTUALENV="$(which virtualenv-3)"
     fi
-
-    ${VIRTUALENV} -p python3 venv && source venv/bin/activate && python3 "$(which pip3)" install pydocstyle
+    if [ $? -eq 1 ]; then
+        # still don't have virtual environment -> use python3 directly
+        python3 -m venv venv && source venv/bin/activate && python3 "$(which pip3)" install pydocstyle
+    else
+        ${VIRTUALENV} -p python3 venv && source venv/bin/activate && python3 "$(which pip3)" install pydocstyle
+    fi
```

Comment:
If virtualenv is a must to run these scripts, why can't install and setup before executing the script instead of changing the script itself?

Ditto to other script changes.

Reply:
The same scripts are run by centos so i just added a hack to reuse this on actions , centos requires virtualenv , but when it comes to github actions there is no virtualenv sp i am using the line : `python3 -m venv venv`

Reply:
@sawood14012 , If you install virtualenv from actions, then these chances shall be avoided right? Something like below?

```
- name Install virtualenv
   run: pip install -U pip && pip install virtualenv
- name: ${{ matrix.scripts }}
      run: bash qa/${{ matrix.scripts }}.sh;
```